tests/components/zwave_js/test_light.py

```diff
@@ -130,7 +130,7 @@ async def test_light(
     assert state.state == STATE_ON
     assert state.attributes[ATTR_COLOR_MODE] == "color_temp"
     assert state.attributes[ATTR_BRIGHTNESS] == 255
-    assert state.attributes[ATTR_COLOR_TEMP] == 370
+    assert state.attributes[ATTR_COLOR_TEMP_KELVIN] == 2702
```

Comment:
Why is it 2702 instead of 2700 as above?

Reply:
2700 above is the "min kelvin" which gets set once
https://github.com/home-assistant/core/blob/72cc1f4d39b2bc844d9e2572f9789c4edd8335d9/homeassistant/components/zwave_js/light.py#L108-L109

2702 is the current value from the integration (looks like a % ratio converted to mireds)
https://github.com/home-assistant/core/blob/72cc1f4d39b2bc844d9e2572f9789c4edd8335d9/homeassistant/components/zwave_js/light.py#L498-L498-500

I see three options:
- the current code is kept with slight rounding discrepencies
- we can change the min/max from `2700`/`6500` to `2702`/`6535` to more closely match the previous defaults
- the formula is changed so the % ratio is based on Kelvin and not Mireds